django_sms_toolkit/utils.py

```diff
@@ -0,0 +1,5 @@
+from .tasks import send_sms as send_sms_task
+
+
+def send_sms(from_number, to_number, body, recipient_id=None):
```

Comment:
With this function we want to change the signature so that `from_number` is optional. 
I suggest the following signature:
```suggestion
def send_sms(to_number, body, recipient_id=None, from_number=None):
```

Inside the function you can set `from_number` from `settings.DJANGO_SMS_TOOLKIT_SETTINGS["DEFAULT_FROM_NUMBER"]` if `from_number is None`

Reply:
Can't we also do it like this:

```suggestion
def send_sms(to_number, body, recipient_id=None, from_number=settings.DJANGO_SMS_TOOLKIT_SETTINGS["DEFAULT_FROM_NUMBER"]):
```